modules/elasticsearch/src/main/java/com/github/jeanbaptistewatenberg/junit5kubernetes/elasticsearch/ElasticSearchPod.java

```diff
@@ -53,39 +48,38 @@ public ElasticSearchPod(final String dockerImageName) {
                 .endContainer()
                 .endSpec()
                 .build());
-        this.waitStrategy = new PodWaitLogStrategy(".*started.*",Duration.ofSeconds(60));
+        this.waitStrategy = new PodWaitLogStrategy(".*started.*", Duration.ofSeconds(60));
     }
 
-    @Override
-    public String getObjectHostIp() {
-        V1Pod v1Pod = this.createdPod.get();
-        if (v1Pod != null) {
-            try {
-                V1Pod retrievedPod = coreV1Api.readNamespacedPod(v1Pod.getMetadata().getName(), NAMESPACE, null, null, null);
-                if (retrievedPod.getStatus() == null) {
-                    throw new RuntimeException("Can't get ip of a non running object.");
-                }
-                return retrievedPod.getStatus().getHostIP();
-            } catch (ApiException e) {
-                throw logAndThrowApiException(e);
-            }
-        } else {
-            throw new RuntimeException("Can't get ip of a non running object.");
-        }
+    public String getHttpHostAddress() {
+        return "http://" + getObjectHostIp() + ":" + getHttpPort();
```

Comment:
In order to match test containers implementation this method should not return the scheme

```suggestion
        return getObjectHostIp() + ":" + getHttpPort();
```